src/components/Header/Header.jsx

```diff
@@ -33,14 +61,27 @@ export class Header extends Component {
     });
   };
 
+  logout = () => {
+    this.props.logout();
+    window.location.href = '/';
```

Comment:
@nkpremices Only set the location once the logout is successful

Reply:
There is no way it can't be successful as it's not consuming any endpoint